File: Grace/C/Grace.c
#include <fcntl.h>
#include <unistd.h>
#include <stdio.h>
#define X "#include <fcntl.h>%c#include <unistd.h>%c#include <stdio.h>%c#define X %c%s%c%c#define C int f=open(%cGrace_output.c%c, O_CREAT | O_WRONLY, 0640);dprintf(f,X,10,10,10,34,X,34,10,34,34,10,10);close(f);%c#define O int m##a##i##n(){C}%cO//this comment complies with the subject"
#define C int f=open("Grace_output.c", O_CREAT | O_WRONLY, 0640);dprintf(f,X,10,10,10,34,X,34,10,34,34,10,10);close(f);
#define O int m##a##i##n(){C}
O//this comment complies with the subject

File: Grace/Python/Grace.py
s='s=%r%sl=%r%sfile=%r%smode=%r%swith open(file,mode) as f:%s    print(s%%(s,l,l,l,file,l,mode,l,l,l,l),file=f);%s    f.close();%s    #mandatory comment'
l='\n'
file='Grace.py'
mode='w'
with open(file,mode) as f:
    print(s%(s,l,l,l,file,l,mode,l,l,l,l),file=f);
    f.close();
    #mandatory comment
